include/libtorrent/hasher.hpp

```diff
@@ -119,6 +122,8 @@ TORRENT_CRYPTO_NAMESPACE
 		gcry_md_hd_t m_context;
 #elif TORRENT_USE_COMMONCRYPTO
 		CC_SHA1_CTX m_context;
+#elif TORRENT_USE_CNG
```

Comment:
the the `TORRENT_CRYPTO_NAMESPACE` above (line 72). The definition of that macro should also depend on whether `TORRENT_USE_CNG` is defined or not. This protects against the ABI effect of using different crypto libraries.

Reply:
thanks, forget to define macro as 0 in case of undefined